src/index.jsx

```diff
@@ -1,10 +1,8 @@
 import React from 'react';
 import {render} from 'react-dom';
 
-class App extends React.Component {
+export default class App extends React.Component {
```

Comment:
ℹ️ This is what was causing the most issues. In order for the tests to run, we need to export this react component and let the other libraries take care of rendering it. I think we got on the track of doing the render in here from looking at tutorials that did not go into depth enough.